com.adilhanney.timing.timing_flutter.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "com.adilhanney.timing.timing_flutter",
```

Comment:
I would just use `com.adilhanney.timing` (granted that the domain is actually yours). But the website at the github repo point to https://adil.hanney.org/timing_flutter/